### PR TITLE
Update zh.js (Quality bump #2)

### PR DESCRIPTION
**Pull request by ItsVenomous**

### CODE REVIEW

**Comment by mty22** [comment]
There seems to be constant changes to Chinese languages.

@0v0katai can you please let me know why? Is there many ways to describe the same thing?

File: zh.js [hunk 1]
```diff
@@ -660,48 +660,48 @@ windows.support.header 支援
...
+streamers.none 当前没有任何Twitch直播!
 error.extentions 如果你有任何浏览器插件，请禁用它们
-error.seek 点此搜索新的游戏
+error.seek 点此寻找新的游戏
 custom.map.upload.limit 上传失败，地图数据大小上限为500kb
 custom.map.missing 请选择至少一种地图
 custom.mode.missing 请选择至少一种游戏模式
 custom.class.missing 请选择至少一种职业
-server.config.maxPlayers 最大玩家数
+server.config.maxPlayers 最多玩家数
```
Comment (0v0katai) [inline zh.js:697]: 最多 (at most) generally refers to an amount (and not number/value) and also is used for comparison, while 最大 straightforwardly means maximum.

File: zh.js [hunk 2]
```diff
@@ -848,26 +848,26 @@ stats.sprays 喷漆数
...
-windows.playerlist.ban 你确定要从此服务器封禁并踢出{0}吗？
+windows.playerlist.ban 你确定要从此房间封禁并踢出{0}吗？
 windows.playerlist.flag 你确定要标记{0}为外挂玩家吗？
 windows.playerlist.unflag 你确定要取消标记{0}为外挂玩家吗？
 windows.playerlist.punish 你确定要对{0}采取措施吗？
 item.view.open 打开模型查看器
 item.view.more 更多信息
 item.sell.cant 此物品不能被出售。
-item.sell.cant.krunker 此物品不能在Krunker内被出售。
+item.sell.cant.krunker 此物品不能被出售。
```
Comment (0v0katai) [inline zh.js:870]: item.sell.cant.krunker Item cant be sold **on krunker**.

**Review by 0v0katai — CHANGES_REQUESTED** [review]
Quite a lot of problems are found, mostly because the suggestions are inaccurate.

File: zh.js [hunk 3]
```diff
@@ -117,13 +117,13 @@ settings.client.colorProfile 色彩配置文件
 settings.client.resourcePacks 资源包
 settings.dev.header 开发者
 settings.dev.forceUpdate 强制刷新皮肤
-settings.dev.forceRegen 总是重生玩家
+settings.dev.forceRegen 强制重生玩家
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:120]: keeping them so they sound the same

File: zh.js (hunk 1 above)
Comment (Kxlici) [reply zh.js:697]: 最多
image

File: zh.js [hunk 4]
```diff
@@ -105,7 +105,7 @@ settings.experimental.mouseAccel 鼠标加速
 settings.experimental.tbInterp TB样条插值
 settings.client.header 客户端
 settings.client.unlimitedFrames 禁用垂直同步
-settings.client.gameCap 游戏捕获
+settings.client.gameCap 游戏录制
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:108]: for me 游戏录制 sounds more clear and easy to understand

File: zh.js [hunk 5]
```diff
@@ -35,10 +35,10 @@ generic.defeat 失败
 generic.victory 胜利
 generic.matchover 比赛结束
 generic.matchaband 比赛被迫终止
-generic.follow 关注
-generic.unfollow 取消关注
+generic.follow 订阅
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:38]: in china we also use 订阅 sometimes

File: zh.js [hunk 6]
```diff
@@ -117,13 +117,13 @@ settings.client.colorProfile 色彩配置文件
...
 settings.dev.forceUpdate 强制刷新皮肤
-settings.dev.forceRegen 总是重生玩家
+settings.dev.forceRegen 强制重生玩家
 settings.dev.extraBan 更多封禁信息
 settings.chat.header 聊天
 settings.chat.chat 显示聊天窗口
 settings.chat.chat.all 总是
 settings.chat.chat.focus 聚焦到[Enter Message]字段时
-settings.chat.messages 显示玩家信息
+settings.chat.messages 显示玩家讯息
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:126]: we do use 讯息

File: zh.js [hunk 7]
```diff
@@ -432,8 +432,8 @@ settings.mods.dot 瞄准镜标线图片
 settings.mods.dot.width 瞄准镜标线图片宽度
 settings.mods.dot.height 瞄准镜标线图片高度
 settings.mods.dot.paste 瞄准镜标线图片网址
-settings.mods.endmessage 比赛结束后自动信息
-settings.mods.profile 头像图片
+settings.mods.endmessage 比赛自动结束讯息
+settings.mods.profile 头像
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:436]: 头像 basically means profile picture/image, i've never heard of 头像图片 in china

File: zh.js [hunk 8]
```diff
@@ -544,14 +544,14 @@ windows.controls.streak4 连杀奖励5
 windows.controls.taunt0 嘲讽（噗嗤声）
 windows.controls.taunt1 嘲讽（口哨声）
 windows.controls.taunt2 嘲讽（放屁声）
-windows.controls.taunt3 嘲讽（模因声）
+windows.controls.taunt3 嘲讽（迷因声）
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:547]: never heard of 模因 in chinese

File: zh.js [hunk 9]
```diff
@@ -132,7 +132,7 @@ settings.chat.chatTimer 聊天文本显示时间
 settings.chat.chatTextOutline 聊天文本轮廓
 settings.chat.chatFilter 脏话过滤（仅限英文）
 settings.quality.header 游戏性能
-settings.quality.updRate 帧率上限
+settings.quality.updRate 限制帧率
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:135]: limit fps is a noun, fps limit is to Limit the fps so for me 限制帧率 is more accurate

File: zh.js [hunk 10]
```diff
@@ -62,19 +62,19 @@ player.waiting 等待更多玩家加入
 player.died 你死了
 player.killed 被此玩家击杀
 player.killstreak {0} on a {1} Kill Streak
-player.inactive 由于你的空闲时间过长，你已被踢出此服务器
+player.inactive 由于你长时间没有活动，你已被踢出此服务器
 player.disconnect 连接已断开
 timer.end 下一场将在{0}后开始
-leaderboard.empty 这里没有任何玩家
-popup.checkpoint 检查点！
+leaderboard.empty 伺服器里没有任何玩家
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:68]: is clearer for me as it is used for old leaderboard in game, that can show better

File: zh.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:38]: > in china we also use 订阅 sometimes

no we don't

[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:38]: > in china we also use 订阅 sometimes

You are carrying out gratuitous duties on him with no solid proof and putting too much unnecessary burden on him. This is too mean bro

File: zh.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:108]: > for me 游戏录制 sounds more clear and easy to understand

游戏录制 既可以指录制的游戏文件 又可说是verb，不准确

File: zh.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:120]: > keeping them so they sound the same

强制mean to "yes or no",总是 mean to sometime always...表示频率
not 100% same.
而且保留这样可以让语言多样化，为什么不呢

File: zh.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:126]: > we do use 讯息

这是简体中文翻译项目啊哥们

File: zh.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:547]: > 



> never heard of 模因 in chinese

Agreement.maybe they are the same

File: zh.js (hunk 9 above)
[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:135]: Translate by minecraft:
E3ZG_2UDYA_6DPX7 JLQDT1

[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:135]: you see,here are both "upd" and "rate"

File: zh.js (hunk 10 above)
[the remark this answers is not in the document]
Comment (SereinP1geon) [reply zh.js:68]: > is clearer for me as it is used for old leaderboard in game, that can show better

bro Let's assume this is right, but why use "伺服器" but not "服务器"？

File: zh.js (hunk 1 above)
Comment (SereinP1geon) [reply zh.js:697]: > 最多 for me does refers to values, and 最大 kinda refers to quantity
> 
> image

最大 =>max =>min
最多 =>most => more

Comment (SereinP1geon) [reply zh.js:697]: I think they mean the same thing.

Comment (SabiGold) [reply zh.js:697]: > 最多 for me does refers to values, and 最大 kinda refers to quantity
> 
> image

I'm a student who major in Chinese Language & Literature, English is not my mother language. The reason why use "最大" is better has two reasons.
1、“最大玩家数” is an abbr for “最大玩家数量”. “数”and“量” usually mean same, but still different. Zheng Xuan's commentary on 《周礼·夏官·量人》 says：“数，多少也；量，长短也。(“数” means “多少”；量 means “长短”)”. This word is a compound word. When we use it, the word that modifies it needs to be restricted by both of the word's roots. So we can only use “最大” that has the widest range of word meanings.
2、“玩家数” is a quantity, quantity uses “大小”；“玩家” is a set, set uses "多少".

File: zh.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (0v0katai) [reply zh.js:547]: Alright, we'll go for 迷因.

File: zh.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (0v0katai) [reply zh.js:38]: > in china we also use 订阅 sometimes

B站UP主不会说「订阅我」，反而是「关注我」，订阅是油管用法

File: zh.js [hunk 11]
```diff
@@ -515,15 +515,15 @@ windows.controls.jump 跳跃
...
+windows.controls.melee 匕首
 windows.controls.equip 拾取
 windows.controls.chat 文字聊天
 windows.controls.voice 语音聊天
 windows.controls.list 玩家列表
 windows.controls.board 排行榜
-windows.controls.interact 交互
-windows.controls.interactSec 次要交互
-windows.controls.confirm 确认交互
+windows.controls.interact 互动
```
[the remark this answers is not in the document]
Comment (0v0katai) [reply zh.js:524]: Actually not really. We do use "互动" when we interact with objects in-game, but here it is a keybind setting. In CSGO it is translated as "交互".

File: zh.js [hunk 12]
```diff
@@ -896,7 +896,7 @@ social.trades.decline 全部拒绝
 social.trades.message 在市场设置中启用「显示交易信息」选项
 social.trades.warning 你将不会在此交易中得到任何物品！
 social.trades.accept 你确定要接受此交易请求吗？
-stats.airdrops 空投
+stats.airdrops 召唤空投次数
```
[the remark this answers is not in the document]
Comment (0v0katai) [reply zh.js:899]: Alright I'll pass this as other stats also have "数" or "次数".

File: zh.js [hunk 13]
```diff
@@ -348,9 +348,9 @@ settings.audio.mic 麦克风音量
 settings.audio.micquality 麦克风音质
 settings.audio.weps 武器音量
 settings.audio.player 玩家音量
-settings.audio.skin 服饰音量
+settings.audio.skin 皮肤音量
 settings.audio.ui 用户界面音量
-settings.audio.asset 地图组件音量
+settings.audio.asset 地图物品音量
```
Comment (0v0katai) [inline zh.js:353]: Asset here means "game assets" which is a general term for all of the components of a map, including objects, background music, sound effects etc. At this moment "组件" has the closest meaning to "assets".

File: zh.js [hunk 14]
```diff
@@ -450,7 +450,7 @@ settings.mods.damage 受伤叠层图片
 settings.mods.damage.paste 受伤叠层图片网址
 settings.mods.overlay 状态栏叠层图片
 settings.mods.overlay.paste 状态栏叠层图片网址
-settings.social.region 地区统计
+settings.social.region 地区数据
```
Comment (0v0katai) [inline zh.js:453]: 数据 means "data" and is quite different from "stats".

File: zh.js [hunk 15]
```diff
@@ -921,25 +921,25 @@ chall.name.bzk 在{1}取得{0}连杀终止击杀
...
+modes.round.loss 此轮不再可胜
+modes.round.switch 已交换双方的初始位置。请做好下一轮的准备
 modes.trade.success 交易成功
 modes.prophunt.limit ［物品猎杀］你已用尽所有物品切换次数
 modes.prophunt.left ［物品猎杀］你还剩余{0}个物品切换次数
 modes.prophunt.rotation.on ［物品猎杀］已启用物品旋转
 modes.prophunt.rotation.off ［物品猎杀］已禁用物品旋转
 modes.prophunt.rotation.reset ［物品猎杀］已重置物品旋转
-commands.login 登录以使用此命令
+commands.login 登录以使用此指令
```
Comment (0v0katai) [inline zh.js:942]: I would say Minecraft (zh_cn) translates "command" as 命令. 指令 here is instead used in zhtr.js just like Minecraft (zh_tw).

**Review by 0v0katai — CHANGES_REQUESTED** [review]

File: zh.js [hunk 16]
```diff
@@ -86,7 +86,7 @@ settings.controls.change 按键设置
 settings.controls.press 按任意键
 settings.controls.leaderboard 游戏内记分板触发类型
 settings.controls.voicechat 语音聊天触发类型
-settings.local.header 本地化 <span id="requiresRestart"><span style="color: #eb5656">*</span> 需要重启游戏</span>
+settings.local.header 本地化 <span id="requiresRestart"><span style="color: #eb5656">*</span> 需重启游戏</span>
```
[the remark this answers is not in the document]
Comment (0v0katai) [reply zh.js]: Changed (by myself)

File: zh.js [hunk 17]
```diff
@@ -508,7 +508,7 @@ windows.controls.right 右
 windows.controls.reload 换弹
 windows.controls.aim1 瞄准
 windows.controls.shoot1 射击
-windows.controls.inspect 检视武器
+windows.controls.inspect 考察武器
```
[the remark this answers is not in the document]
Comment (0v0katai) [reply zh.js]: > can we keep it as 展示武器全貌？

Nope, first 展示武器全貌 is redundant, second you cannot inspect the weapon **fully** and thus cannot say “全貌” (whole appearance), and third CSGO translates it as 检视武器.

File: zh.js [hunk 18]
```diff
@@ -923,16 +923,16 @@ chall.name.rk 在{1}取得{0}复仇击杀
 event.doubleXP 已为此场比赛启用双倍职业经验
 ranked.join 已加入排位赛。罢赛将会受到暂时性禁赛惩罚
 ranked.progress 比赛进行中
-ranked.team 请加入队伍
+ranked.team 请加入一个队伍
```
Comment (0v0katai) [inline zh.js:926]: “一个”在这里显得有些多余。而且如果加上的话，这带来的隐藏意思是玩家能选择任何一个队伍去加入，但是这在Faceit排位赛中并不适用，因为队伍已经定下来了，他们不能加入其他队伍

File: zh.js [hunk 19]
```diff
@@ -923,16 +923,16 @@ chall.name.rk 在{1}取得{0}复仇击杀
...
 ranked.switch 回合开始后无法换枪
 ranked.spectate 排位赛中禁止观战
 ranked.login 登录以进入比赛
 ranked.sync 缺失同步数据
 ranked.team.spawn 请先加入队伍再进入比赛
 ranked.class.spawn 请先选择职业再进入比赛
-modes.round.win 你的队伍已赢下此回合
-modes.round.loss 此回合不再可胜
-modes.round.switch 已交换双方的初始位置。请做好下一回合的准备
+modes.round.win 你的队伍已赢下此轮
```
Comment (0v0katai) [inline zh.js:933]: 轮和回合意思一样，而且上面第927行已经用了“回合”。

**Review by 0v0katai — CHANGES_REQUESTED** [review]

**Review by 0v0katai — APPROVED** [review]
@mty22 Good to go

**Comment by mty22** [comment]
thanks! can everyone please mention their IGN's here

**Comment by ItsVenomous** [comment]
ign itsVenomous

**Comment by 0v0katai** [comment]
0v0katai

**Review by 0v0katai — APPROVED** [review]

**Pull request merged**